app/src/ui/app.tsx

```diff
@@ -147,6 +147,10 @@ export class App extends React.Component<IAppProps, IAppState> {
         },
         { timeout: ReadyDelay }
       )
+      const interval = 1000 * 60 * 5
```

Comment:
:art: let's extract this up to a const with a more descriptive name.

For example, we have this defined earlier in the file:


```
/** The interval at which we should check for updates. */
const UpdateCheckInterval = 1000 * 60 * 60 * 4
```